Core/src/IfcGeometryConverter/GeometrySettings.h

```diff
@@ -164,6 +164,44 @@ namespace OpenInfraPlatform
 					}
 				}
 
+				bool areEqual(const double first,const double second) const
+				{
+					if (abs(first - second) < getPrecision()) 
+					{
+						return true;
+					}
+					return false;
+				}
+
+				bool areEqual(const carve::geom::vector<2>& first, const  carve::geom::vector<2>& second) const
+				{
+					if (abs(first.x - second.x) < getPrecision() && abs(first.y - second.y) < getPrecision())
+					{
+						return true;
+					}
+					return false;
+				}
+				
+				bool areEqual(const  carve::geom::vector<3>& first, const  carve::geom::vector<3>& second) const
+				{
+					if (abs(first.x - second.x) < getPrecision() && abs(first.y - second.y) < getPrecision() && abs(first.z - second.z) < getPrecision())

```

Comment:
Similarly, call the other overload.

Reply:
Maybe carve itself has some functionality which can be used? Did you investigate?

Reply:
Resolved: 1b84b1d